tor/archiver.py

```diff
@@ -0,0 +1,82 @@
+import sys
+import logging
+
+from praw import Reddit
+
+from tor import config
+from tor.core.initialize import configure_logging
+from tor.core.initialize import configure_tor
+from tor.helpers.misc import explode_gracefully, subreddit_from_url
+
+from tor.strings.urls import reddit_url
+
+from time import sleep
+from datetime import datetime
+
+
+def run(tor, config, archive):
+    # TODO the bot will now check ALL posts on the subreddit.
+    # when we remove old transcription requests, there aren't too many left.
+    # but we should make it stop after a certain amount of time anyway
+    # eg. if it encounters a post >36 hours old, it will break the loop
+
+    # TODO we can use .submissions(end=unixtime) apparently
+
+    for post in tor.new():
+        # [META] - do nothing
+        # [UNCLAIMED] - remove
+        # [COMPLETED] - remove and x-post to r/tor_archive
+        # [IN PROGRESS] - do nothing (should discuss)
+        flair = post.link_flair_css_class
+
+        if flair not in ('unclaimed', 'completed'):
```

Comment:
I'm comparing CSS class rather than flair text, so there's no need to look it up in the flair dict.